openshift-with-appstudio-test/e2e/util.go

```diff
@@ -276,3 +278,58 @@ func streamFileYamlToTektonObj(path string, obj runtime.Object, ta *testArgs) ru
 	}
 	return decodeBytesToTektonObjbytes(bytes, obj, ta)
 }
+
+func createAppstudioApp(ta *testArgs) {
+	name := generateName("test-application-")
+	app := &applicationservice.Application{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ta.ns,
+		},
+		Spec: applicationservice.ApplicationSpec{
+			DisplayName: name,
+		},
+	}
+
+	if err := crClient.Create(context.Background(), app); err != nil {
+		debugAndFailTest(ta, err.Error())
+	}
+
+	ta.appstudioApp = app
+
+}
+
+func createAppstudioComponent(ta *testArgs) {
+	name := generateName("test-component-")
+
+	quayOrg := os.Getenv("QUAY_E2E_ORGANIZATION")
```

Comment:
should we update the README at https://github.com/redhat-appstudio/jvm-build-service/blob/main/.ci/README.md to describe this env var ?

Reply:
Good point.

This env var is required if your quay.io account doesn't have permissions to push to `quay.io/redhat-appstudio/test-images` repo - in that case the test would be failing on the `build-container` taskrun if you'd try to run the test locally.

I think it's not needed to document it in the .ci folder (since CI will always work with `redhat-appstudio-qe` quay and github orgs by default), but I think it'd make sense to add a note about it to the `openshift-with-appstudio-test/e2e/`. Or perhaps in the root readme file? wdyt?

Reply:
yeah sure create a README in https://github.com/redhat-appstudio/jvm-build-service/tree/main/openshift-with-appstudio-test/e2e and add this bit there, along with the basic need an openshift cluster with appstudio bootstrapped, kubeconfig set or cluster login

thanks

Reply:
created a readme file: https://github.com/redhat-appstudio/jvm-build-service/pull/161/commits/724fc86081912ebcacedeea7697eab8f81f13e03